.github/workflows/build-and-test-Linux.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Build CBMC tools
         run: |
           make -C src minisat2-download
-          make -C src CXX='ccache /usr/bin/g++' cbmc.dir goto-cc.dir goto-diff.dir -j2
```

Comment:
Why did the compiler change here?

Reply:
This is to make sure we can actually use the compiler cache (and don't trash it by putting a different configuration in there).